CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Avoid page timeout by skipping HGVS validations in ClinVar multistep submission for non-MANE transcripts from variants in build 38 (#5302)
 - Sashimi view page displaying an error message when Ensembl REST API (LiftOver) is not available (#5322)
 - Refactored the liftover functionality to avoid using the old Ensembl REST API (#5326)
+- Documentation available in dark mode (#5343)
```

Comment:
This is not really a fix, more like something added? Also mention that you improved the software installation instructions (this is more like a change)